docs/index.md

```diff
@@ -232,7 +278,8 @@ Open your projects `/ios/Podfile` and add any of the globals shown below to the
 
 ```ruby
 # Override Firebase SDK Version
-$FirebaseSDKVersion = '10.3.0'
+# Between `use_react_native!()` paragraph and `target 'RNFBTests' do ... end`
+$FirebaseSDKVersion = '10.4.0'
```

Comment:
As mentioned in another comment we are not compatible with 10.4.0 yet. This is an advanced configuration method anyway and basically no one should use it, definitely no one should use it long-term, as react-native-firebase should control the dependency version.

```suggestion
$FirebaseSDKVersion = '10.3.0'
```

Reply:
yeah I checked it also works with 10.3.0 too
thanks 👍